apis/fluentd/v1alpha1/tests/expected/fluentd-mixed-cfgs-multi-tenant-output.cfg

```diff
@@ -7,41 +7,41 @@
   @id  main
   @type  label_router
   <route>
-    @label  @c82a1bf977b4129a24c9783e1c196183
+    @label  @b03d7a71344b3738580b3c8ef54d7a75
     <match>
-      namespaces  kube-system,kubesphere-system
+      namespaces  default,kube-system
     </match>
   </route>
   <route>
-    @label  @aec71d90c93884d7411a6830b4fb6d47
+    @label  @a504500266de68fa800121c2f3c9088e
     <match>
-      namespaces  kubesphere-logging-system
+      namespaces  fluent
     </match>
   </route>
 </match>
-<label @c82a1bf977b4129a24c9783e1c196183>
+<label @b03d7a71344b3738580b3c8ef54d7a75>
   <match **>
     @id  ClusterFluentdConfig-cluster-fluentd-config-cluster::cluster::clusteroutput::fluentd-output-cluster-0
     @type  elasticsearch
-    host  elasticsearch-logging-data.kubesphere-logging-system.svc
+    host  elasticsearch-logging-data.fluent.svc
     logstash_format  true
     logstash_prefix  ks-logstash-log
     port  9200
   </match>
 </label>
-<label @aec71d90c93884d7411a6830b4fb6d47>
+<label @a504500266de68fa800121c2f3c9088e>
   <match **>
-    @id  FluentdConfig-kubesphere-logging-system-fluentd-config-user1::kubesphere-logging-system::output::fluentd-output-user1-0
+    @id  FluentdConfig-fluent-fluentd-config-user1::fluent::output::fluentd-output-user1-0
     @type  elasticsearch
-    host  elasticsearch-logging-data.kubesphere-logging-system.svc
+    host  elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc